### DISCUSSION TITLE
Feature Suggestion: 24 Hour Price Change Display

### DISCUSSION THREAD

**Discussion by Puffball101961** [taoteh1221/Slideshow_Crypto_Ticker#48] [Ideas]
Hi, I have a feature suggestion to add a 24 hour price change display, just above the trading volume display, or to have an option in the config to replace the volume display with a 24 hour change volume. Not sure if this is feasible or not, just putting the idea out there.

**Comment by taoteh1221** [comment]
Hello. Thanks for the feedback, I appreciate it. It's definitely very feasible, in fact those exact features have already been on the TODO list here (under the "FEATURES / STRUCTURING SECTION"): https://raw.githubusercontent.com/taoteh1221/Slideshow_Crypto_Ticker/main/DOCUMENTATION-ETC/TODO.txt

I'm not sure when I will put some time aside to do this, but it will happen eventually.

**Comment by Puffball101961** [comment]
Hi, sorry I didn't see that todo document. I was also thinking about trying to implement this myself, so if I do I will be sure to open a PR

**Reply by taoteh1221** [reply]
No worries! :) Cool. If you do decide to do a PR, check out the "CONTRIBUTION GUIDELINES (FOR DEVELOPERS)" section located here: https://raw.githubusercontent.com/taoteh1221/Slideshow_Crypto_Ticker/main/CONTRIBUTING.txt
